lesson16/index.js

```diff
@@ -0,0 +1,124 @@
+// const url = "https://myjson.dit.upm.es/api/bins/hyv9";
+const url = "https://myjson.dit.upm.es/api/bins/d9vx";
+const newsBlock = document.getElementById("news");
+const newsTabs = document.getElementById("news_Tabs");
+
+const createNewsTabs = (allData) => {
+  allData.forEach((content, index) => {
+    const li = document.createElement("li");
+    li.classList.add("tab_item");
+    li.id = content.category;
+    li.textContent = content.category;
+    newsTabs.appendChild(li);
+    if(content.isActive === true){
+      li.classList.add("is-active");
+    }
```

Comment:
IMO
isActive is a boolean.

```suggestion
    if(content.isActive){
      li.classList.add("is-active");
    }
```
or 

```js
 content.isActive && li.classList.add("is-active");
```
It can also be written like this
This is nits.

Reply:
[f8965ed](https://github.com/hippopo-moo/js-handson-work/pull/18/commits/f8965ed425e0460a137d88f58b0b54b3235cc29c)

I learned a lot from knowing that there is such a way to describe it. So I rewrote it.
Thank you very much!